ios/BurntModule.swift

```diff
@@ -21,10 +21,19 @@ enum AlertPreset: String, Enumerable {
       case .spinner:
         return .spinner
       case .custom:
-        guard let image = UIImage.init( systemName: options?.icon?.name ?? "swift") else {
-          throw BurntError.invalidSystemName
+        let weight: UIImage.SymbolWeight
```

Comment:
what about: 

```swift
let weight = try? toFontWeight(from: options?.icon?.fontWeight) ?? .regular
```

Reply:
> that being said, this worked. wdyt?

that code doesn't not work because it will throw if weight isn't given, but it's optional

Reply:
yes, i ran the exact code, here's what the whole custom block looks like:
```
      case .custom:
        let weight = try? toFontWeight(from: options?.icon?.fontWeight) ?? .regular

        let configuration = UIImage.SymbolConfiguration(weight: weight)
         guard let image = UIImage(systemName: options?.icon?.name ?? "swift", withConfiguration: configuration) else {
            throw BurntError.invalidSystemName
        }

        return .custom(image.withTintColor(options?.icon?.color ?? .systemBlue, renderingMode: .alwaysOriginal))
```

with the same error (i just tried again)